examples/Attributes/publications.md

```diff
@@ -33,30 +23,72 @@ publications:
   range: publication
 ```
 
-Note, the range for `biolink:publications` is `biolink:publication` which is actually a [Biolink Model class](https://biolink.github.io/biolink-model/docs/Publication.html#class-publication).  However, using a collection of CURIEs as shorthand syntax for representing a full list of `biolink:Publication` objects 
-is expected in a TRAPI message (see below).
+### Implementation Guidance
 
+1. The `biolink:publications` edge property MUST be used as the `attribute_type_id` to report 
+publications that support an Edge.  Each reported edge MUST have at least one Attribute object with 
```

Comment:
> "Each reported edge MUST have at least one Attribute object with `attribute_type_id`:`biolink:publications`" 

I don't think we want to require that every Edge MUST have a publication Attribute. There are many that just don't have one.